modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -280,7 +280,10 @@ public FluoConfiguration setApplicationName(String applicationName) {
     setProperty(CONNECTION_APPLICATION_NAME_PROP, applicationName);
     return this;
   }
-
+  /**
+   * Returns the application name.
```

Comment:
@kpm1985 I think the docs you added are nice.  In addition to this I was looking for something like the following for #900

 ```
<p>
  Gets the value of the property {@value #CONNECTION_APPLICATION_NAME_PROP}
```

Note in this case I do not think its necessary to mention `CLIENT_APPLICATION_NAME_PROP` as that is a deprecated property.